packages/react-components/react-rating/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@fluentui/react-rating-preview",
-  "version": "0.4.0",
+  "name": "@fluentui/react-rating",
+  "version": "9.0.0-alpha.0",
```

Comment:
Shouldn't this just be `9.0.0`?

Reply:
Yea, this was the autogenerated version number from the script